common/src/main/java/com/wynntils/features/user/tooltips/ItemStatInfoFeature.java

```diff
@@ -58,14 +65,20 @@ public class ItemStatInfoFeature extends UserFeature {
     public float obfuscationChanceEnd = 0.04f;
 
     @Config
-    public boolean reorderIdentifications = true;
+    public StatListOrdering identificationsOrdering = StatListOrdering.DEFAULT;
 
     @Config
     public boolean groupIdentifications = true;
 
+    @Config
+    public boolean identificationDecorations = true;
+
     @Config
     public boolean overallPercentageInName = true;
 
+    @Config
+    public boolean showBestValueLastAlways = true;
```

Comment:
I have no idea what this does. But I assume we have not changed behaviour from before? Or did we?

Reply:
I mostly figured this out from i18n.

Reply:
In Wynncraft vanilla, there is no clear consistency which value is put where in a range, so "X to Y" could mean either that X is the worst possible value (and Y is the best), or that Y is the worst possible value (and X is the best).

If this is false, we will copy Wynncraft behavior. If this is true, we will always go from "worst" to "best".

The problems arise when you have like spell costs which has negative values (but "more negative" is better), and where a "positive" stat has a negative value (like -5 health regen). Iirc, Wynntils also makes this different depending on whether an item is in the Trade Market or in your inventory. This was too complex for me to handle to replicate, so I picked one of the two methods to use.

Reply:
isn't this exclusively the case with spell cost IDs, thanks to the way they were recently changed? i'm not sure if it's worth worrying about replicating what is effectively a bug on wynn's end